lib/classes/Variables.js

```diff
@@ -822,6 +827,17 @@ class Variables {
       });
   }
 
+  getValueStrToBool(variableString) {
+    const groups = variableString.match(this.strToBoolRefSyntax);
```

Comment:
Are we sure that in 100% cases `variableString` will be type of string? What if it comes from a source that returns a number? 

@pmuens you have more experience with that. What do you think?

Reply:
Yes, that's a good point. Right now a `variableString` looks something like this: `cf:test-stack.testOutput` or `ssm:/test/path/to/param`, so it seems like it's always a string (at least after looking at the unit tests).

However it could be the case that this changes in the future (not entirely sure why, but if we bloat the Variables implementation we might run into such situation).